templates/sync-workspace-secret.yaml

```
@@ -0,0 +1,10 @@
+{{- if .Values.CACerts }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cacert
```

The secret should be in the same namespace as the rest of the resources.
```suggestion
  name: cacert
  namespace: {{ .Release.Namespace }}
```